scripts/inputs.sql

```diff
@@ -0,0 +1,17 @@
+INSERT INTO hospital VALUES(1, 'Saint Johns Hospital', 200),
```

Comment:
Added the commands used to insert records

Reply:
Well, the `SERIAL` type is auto incrementing, so no need to specify it for new records. Nevermind, take note of this next time. 
```suggestion
INSERT INTO hospital(name, max_patient_amount) VALUES('Saint Johns Hospital', 200),
```